src/main/java/pro/quicksense/modules/common/Result.java

```diff
@@ -0,0 +1,57 @@
+package pro.quicksense.modules.common;
+
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+public class Result<T> implements Serializable {
```

Comment:
It seems like we don't need to reinvent the wheel; the Spring Boot framework provides out-of-the-box methods.

example:  https://blog.51cto.com/u_16213382/10361180